core/src/test/scala/unit/kafka/integration/KafkaServerTestHarness.scala

```diff
@@ -260,9 +274,12 @@ abstract class KafkaServerTestHarness extends QuorumTestHarness {
   }
 
   def killBroker(index: Int): Unit = {
```

Comment:
maybe we should keep origin implementation since it expect to await shutdown.

Reply:
My point was "should we keep `_brokers(index).awaitShutdown()`"? It seems the latch used by `awaitShutdown` should be notified after `shutdown` get completed, so maybe it is fine to remove it. It would be great to add comments if you prefer to remove `awaitShutdown`

Reply:
Oh I see. It was a mistake to remove `awaitShutdown`. I think we should leave the behavior as is, I don't see any reason why the new `killBroker(idx, timeout)` alternative shouldn't also call `awaitShutdown`.